models/public/deblurgan-v2/accuracy-check.yml

```diff
@@ -0,0 +1,62 @@
+models:
+  - name: deblurgan-v2
+
+    launchers:
+      - framework: onnx_runtime
+        model: deblurgan-v2.onnx
+        adapter:
+          type: image_processing
+          mean: 127.5
+          std: 127.5
+
+    datasets:
+      - name: GoPro
+        preprocessing:
+          - type: bgr_to_rgb
+          - type: normalization
+            mean: 127.5
+            std: 127.5
+          - type: padding
+            dst_height: 736
+            dst_width: 1312
+            use_numpy: True
+            pad_type: right_bottom
+        postprocessing:
+          - type: corner_crop_image
+            apply_to: prediction
+        metrics:
+          - type: psnr
+            scale_border: 0
+            presenter: print_vector
+          - type: ssim
+            presenter: print_vector
+
+  - name: deblurgan-v2
+
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: image_processing
+          mean: 127.5
+          std: 127.5
```

Comment:
please note that PSNR sensitive to channels order, you need to align output image channels order (BGR) with gt channels order (RGB) using `reverse_channels: True` in adapter

Reply:
Parameters of adapter are right, output image channels order is RGB.